ext_localconf.php

```diff
@@ -13,7 +12,25 @@
 $GLOBALS['TYPO3_CONF_VARS']['BE']['flexFormXMLincludeDiffBase'] = true;
 
 $renderFceHeader = '';
-if ($_EXTCONF['enable.']['renderFCEHeader']) {
+
+if (version_compare(TYPO3_version, '9.5.0', '>=')) {
+    try {
+        $backendConfiguration = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(
+            \TYPO3\CMS\Core\Configuration\ExtensionConfiguration::class
+        )->get('templavoilaplus');
+    } catch (
+    \TYPO3\CMS\Core\Configuration\Exception\ExtensionConfigurationExtensionNotConfiguredException
+    |\TYPO3\CMS\Core\Configuration\Exception\ExtensionConfigurationPathDoesNotExistException $e
+    ) {
+        $backendConfiguration = [];
+    }
+} else {
+    $backendConfiguration = unserialize(
+        $GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['templavoilaplus'],
+        ['allowed_classes' => false]
+    );
+}
+if (isset($backendConfiguration['enable.']['renderFCEHeader']) && $backendConfiguration['enable.']['renderFCEHeader']) {
```

Comment:
In 9.x and above the dots where removed IMHO (see Tvp\TemplaVoilaPlus\Service\ConfigurationService::removeDotsFromArrayKeysRecursive

Reply:
copy and paste error, sry